istio-interop-testing/src/main/java/io/grpc/testing/istio/EchoTestServer.java

```diff
@@ -114,10 +126,10 @@ static Map<String, List<String>> preprocessArgs(String[] args) {
     return ImmutableMap.<String, List<String>>builder().putAll(argsMap).build();
   }
 
-  /** Turn gRPC ports from a string list to an int list. */
+  /** Turn ports from a string list to an int list. */
   @VisibleForTesting
-  static List<Integer> getGrpcPorts(Map<String, List<String>> args) {
-    List<String> grpcPorts = args.get("--grpc");
+  static List<Integer> getPortsList(Map<String, List<String>> args, String flagName) {
+    List<String> grpcPorts = args.get(flagName);
```

Comment:
Looking at the test for `processedArgs.get("--port")` - ports are expected to be in quotes:

```java
    assertEquals(
        processedArgs.get("--port"),
        ImmutableList.of(
            "\"18080\"",
            "\"18085\"",
            "\"18443\"",
            "\"18081\"",
            "\"19443\"",
            "\"18082\"",
            "\"18084\"",
            "\"18083\"",
            "\"8080\"",
            "\"3333\""));
  }
```

This method does `Integer.parseInt()` - this will throw an `java.lang.NumberFormatException:`, right? I wonder why tests didn't catch it. Are we missing `getPortsList` test? Or I didn't notice where quotes are stripped?

Reply:
Good catch! Fixed!